src/styles/utils/_variables.scss

```diff
@@ -0,0 +1,9 @@
+$watch-size: 700px;
+$center-size: 50px;
+$hands-width: 15px;
+$half-hend-width: $hands-width / 2;
+$seconds-hand-langht: $watch-size / 2 - 50px;
+$seconds-hand-top: -$seconds-hand-langht + $center-size / 2;
+$hand-left: $center-size / 2 - $half-hend-width;
+$minutes-hand-lenght: $seconds-hand-langht - 50px;
```

Comment:
There's a typo in the variable name, it should be `$minutes-hand-length` instead of `$minutes-hand-lenght`. Please correct the variable name.